config/routes.rb

```diff
@@ -31,8 +29,14 @@
   resources :merchants, only: [:index]
 
   resources :users, only: [:create]
-  
+
+  resources :carts, only: [:create]
```

Comment:
Since you're creating the resources here, you probably don't need to have lines 39-41.